packages/react-tabs/src/next/components/Tabs/TabItem.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { getNativeProps, divProperties, initializeComponentRef } from '@fluentui/utilities';
+import { TabItemProps } from './TabItem.types';
+
+// const COMPONENT_NAME = 'TabItem';
+
+export class TabItem extends React.Component<TabItemProps, {}> {
```

Comment:
Sorry if I missed some discussion here but is there any way we can avoid entirely duplicating the code? The duplication almost guarantees it will get out of sync and bug fixes will be missed in one version or the other.

Reply:
Pivot is going to be deprecated, and Tabs is going to be further modified after this PR for convergence work. In the interim, Pivot needs to continue to work without breaking changes. There are two basic approaches:
1. Rename Pivot to Tabs, and keep Pivot as a "thin" compat layer that accounts for all of the breaking changes.
   * The compat layer can get complicated, and is its own engineering problem. For example, Pivot uses old CommandButton for its tab buttons, and exposes that through its API via TabItem.headerButtonProps. If we replace that with the new Button, we need to keep headerButtonProps working. Possibly solvable, but potentially a lot of extra work.
2. Fork Pivot to /next/Tabs, and do the API breaking convergence work on the Tabs. In the future, Pivot will be deprecated and moved to /compat. Any new features and updates will be added to Tabs, not Pivot.

I was originally going to do 1, but after talking to Justin and David, decided to do 2, which is what this PR is.

Reply:
Got it, sorry. This would be a lot less confusing if we were in the office. :(